docs/guide/object-backgrounds.md

```diff
@@ -42,11 +53,11 @@ TexturedBackground {
 
 ### TexturedBackground Field Summary
 
-- `texture`: Defines the texture of the background.
+- `texture`: Defines the texture of the background. This field accepts the following values: `"dawn_cloudy_empty"`, `"dusk"`, `"empty_office"`, `"entrance_hall"`, `"factory"`, `"mars"`, `"morning_cloudy_empty"`, `"noon_building_overcast"`, `"noon_cloudy_countryside"`, `"noon_cloudy_empty"`, `"noon_cloudy_mountains"`, `"noon_park_empty"`, `"noon_stormy_empty"`, `"noon_sunny_empty"`, `"noon_sunny_garden"`, `"twilight_cloudy_empty"` and `"mountains"`.
 
 - `skybox`: Specifies if the `texture` field is used to define the skybox shown in the scene background.
 
-- `reflections`: Specifies if the `texture` field is used in the reflections of the [PBRAppearance](../reference/pbrappearance.md) nodes.
+- `reflections`: Specifies if the `texture` field is used in the reflections of the PBRAppearance nodes.
```

Comment:
Why did you remove this link?

Reply:
Because the link was added manually, which is wrong, we should not edit these files at all as they are generated.
The link can be added in the description of the field in the PROTO itself, but IMO it doesn't really make sense.

Reply:
I see. It would be nice however, that our python script generates automatically this kind of links based on a dictionary of nodes.

Reply:
Yes, I will open a new issue about this.

Reply:
Finally, I did add this feature in this PR as it was quite easy, it adds several links previously not present.